ansible/roles/common/defaults/main.yaml

```diff
@@ -6,3 +6,13 @@ bin_dir: /usr/bin
 # The directory used by Ansible to temporarily store
 # files on Ansible managed systems.
 ansible_temp_dir: /tmp/.ansible/files
+
+# The directory used for the docker config
+system_config_dir: "/etc/sysconfig"
```

Comment:
ah nice call, at least I did follow up on the plan and it did get merged... removed it from here